CustomScript/HandlerManifest.json

```diff
@@ -1,13 +1,17 @@
-[{
-  "name": "CustomScript",
-  "version": "1.5.1.0",
-  "handlerManifest": {
-    "installCommand": "customscript.py -install",
-    "uninstallCommand": "customscript.py -uninstall",
-    "updateCommand": "customscript.py -update",
-    "enableCommand": "customscript.py -enable",
-    "disableCommand": "customscript.py -disable",
-    "rebootAfterInstall": false,
-    "reportHeartbeat": false
+[
+  {
+    "namespace": "Microsoft.OSTCExtensions",
+    "name": "CustomScriptForLinux",
+    "extension_version": "1.5.2.0",
```

Comment:
The CI tool needs to know the version that you're deploying, and I need to store that metadata somewhere.  I can store it here, manifest.xml, or some other document.